nixos/modules/config/pulseaudio.nix

```diff
@@ -135,6 +177,7 @@ in {
 
     (mkIf cfg.enable {
       environment.systemPackages = [ cfg.package ];
+      environment.sessionVariables = moduleEnvVars;
```

Comment:
I'm not sure what you mean here exactly- the module packages have no binaries, and these variables just tell pulseaudio where to look for the module configuration. If you mean wrapping pulseaudio itself to look for moduleEnvVars wouldn't that require building pulseaudio each change?